testing/autofs/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# Contributor: Alex Laskin <alex@lask.in>
+# Maintainer: Alex Laskin <alex@lask.in>
+pkgname=autofs
+pkgver=5.1.4
+pkgrel=0
+pkgdesc="A kernel-based automounter for Linux"
+url="https://www.kernel.org/pub/linux/daemons/autofs/"
+arch="all"
+license="GPL"
```

Comment:
Use one of the [SPDX](https://spdx.org/licenses/) license descriptions, for example `GPL-2.0-only`

Reply:
Ok, sounds reasonable.